lib/hal/rpihal.ex

```diff
@@ -30,7 +31,7 @@ defmodule Inky.RpiHAL do
 
   @impl HAL
   def init(args) do
-    display = Map.fetch!(args, :display)
+    display = args[:display] || raise(ArgumentError, message: ":display missing")
```

Comment:
Is the message based off of any particular pattern? Seems terse.
":display missing in init args" or something to that effect if we need a custom error at all would be my first guess. But  haven't raised a lot so far.

Reply:
Hm. Good point about the message, I'll add `in init args`.